src/core/symbology/qgsellipsesymbollayer.cpp

```diff
@@ -272,11 +273,11 @@ void QgsEllipseSymbolLayer::renderPoint( QPointF point, QgsSymbolRenderContext &
     return;
   }
 
-  QMatrix transform;
+  QMatrix4x4 transform;
```

Comment:
Couldn't this just be a QTransform?

Reply:
Let me quote myself:

> I don't know what I'm doing and completely bet on unit tests to prove me wrong

so ... possibly :stuck_out_tongue_closed_eyes:

Reply:
Let me quote myself:

> I don't know what I'm doing and completely bet on unit tests to prove me wrong

so ... possibly :stuck_out_tongue_closed_eyes:

Reply:
I'm 90% sure you could just change the variable to QTransform and it'll work correctly

Reply:
Makes a lot of sense, patch inbound